examples/ping1d-servo/ping1d-servo.ino

```diff
@@ -0,0 +1,112 @@
+/**
+ *  This example is targeted toward the arduino platform
+ *
+ *  This example demonstrates basic usage of the Blue Robotics Ping1D c++ API
+ *  together with a servo-controlled device.
+ *
+ *  Context for the example is in this forum thread:
+ *  https://discuss.bluerobotics.com/t/using-ping1d-and-servo-with-arduino-nano-every/11715
+ *
+ *  Communication is performed with a Blue Robotics Ping1D Echosounder
+ *  Includes servo-style control of a device (e.g. a servo motor)
+ */
+
+#include <Servo.h>
+#include "ping1d.h"
+#include "SoftwareSerial.h"
+
+/* NOTE: Servo and SoftwareSerial clash!
+ *   Both libraries use the same timer.
+ *   This code avoids clashes by only keeping the servo 'attach'ed while moving.
+ *   If the servo is under load, undesired movement may occur while using Ping.
+ */
+
+/* Ping Sonar */
+void readSonar(bool display=true); // declare readSonar function
+static const uint8_t arduinoRxPin = 9;
+static const uint8_t arduinoTxPin = 10;
+SoftwareSerial pingSerial = SoftwareSerial(arduinoRxPin, arduinoTxPin);
+static Ping1D ping { pingSerial };
+// speed of sound in operating medium (~343 m/s for air, ~1500 m/s for seawater)
+static const uint32_t v_sound = 343000; // mm/s
+
+/* Servo */
+void moveServo(int angle, bool display=true); // declare moveServo function
+Servo myservo;
+// angles, in degrees
+int pos = 0;
+int max_angle;
+int min_angle = 0;
```

Comment:
There is a mix of camelCase and snake_case here, check the others examples for consistency.